__tests__/app.test.js

```diff
@@ -31,9 +33,560 @@ describe("NCNews-Server Unit Tests", () => {
+                        }});
+                    });
+                });
+
+                test(`[ 200 ] Makes sure this endpoint works if URL is formatted slightly differently`, () => {
+
+                    return request(app).get(`/api/articles/?order=asc`).expect(200).then((response) => {
+                        const articlesArray = response.body.articles;   
+                        expect(articlesArray).toHaveLength(12);
+        
+                        articlesArray.forEach((e, index) => {
+                            if (index < articlesArray.length - 1) {
+                            expect(Date.parse(articlesArray[index][`created_at`]) < Date.parse(articlesArray[index + 1][`created_at`])).toBe(true);
+                        }});
+                    });
+                });
+
+                test(`[ 200 ] When passed "desc" as a URL parameter, server responds with an array of objects, where the objects are sorted by Date using the 'created_at' property in Descending order.`, () => {
+
+                    return request(app).get(`/api/articles?order=desc`).expect(200).then((response) => {
+                        const articlesArray = response.body.articles;   
+                        expect(articlesArray).toHaveLength(12);
+        
+                        articlesArray.forEach((e, index) => {
+                            if (index < articlesArray.length - 1) {
+                            expect(Date.parse(articlesArray[index][`created_at`]) > Date.parse(articlesArray[index + 1][`created_at`])).toBe(true);
+                        }});
+                    });
+                });
+
+                test(`[ 404 ] Responds with an error when passed invalid parameters, test 1`, () => {
+
+                    return request(app).get(`/api/articles?order=sdfdefds`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+
+                test(`[ 404 ] Responds with an error when passed invalid parameters, test 2`, () => {
+
+                    return request(app).get(`/api/articles?order=1337`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+        
+                test(`[ 404 ] Responds with an error when passed an SQL injection test 1`, () => {
+                    
+                    return request(app).get(`/api/articles?order=1&#59;&nbsp;DROP&nbsp;TABLE&nbsp;articles`).expect(404).then((response) => {
+                        expect(response.body).toEqual({ error: "Error 404! File Not Found" });
+                    });
+        
+                });
+        
+                test(`[ 404 ] Responds with an error when passed an SQL injection test 2`, () => {
```

Comment:
Again this test is redundant it will have passed right away, Side note I dont think the error message of file not found is particularly helpful here, were not looking for files